lib/assets/javascripts/utils/tinymce.js

```diff
@@ -36,6 +37,7 @@ export const defaultOptions = {
   paste_remove_styles_if_webkit: true,
   paste_remove_spans: true,
   paste_strip_class_attributes: 'all',
+  max_height: 350,
```

Comment:
Perhaps this is solving the problem but the root cause for seeing a huge editor that is resized automatically later on is due to using display:none instead of visibility:hidden to hide a tinymce editor.